### DISCUSSION TITLE
Remove redundant amount of map background usage in song select

### DISCUSSION THREAD

**Discussion by nobully-osu** [ppy/osu#25817] [Ideas] [answered]
Perhaps this is more of a change request rather than a feature request, but I feel the map's background being visible in 3 separate places on the song select is overkill. Removing or adding a toggle to disable the background on the map bar itself (the one you click to see the difficulties of a mapset) would make this feel better. 

Alternatively having something similar to how stable does thumbnails would be a bit better (also with the ability to toggle, like stable).

**Comment by peppy** [comment, marked as answer]
We're redesigning song select.

**Reply by nobully-osu** [reply]
Sweet!
